.circleci/config.yml

```diff
@@ -100,24 +99,24 @@ jobs:
   test-python:
     working_directory: ~/metadeploy
     docker:
-      - image: circleci/python:3.8.3
+      - image: circleci/python:3.8
         environment:
           BUCKETEER_AWS_ACCESS_KEY_ID: circle_test
           BUCKETEER_AWS_SECRET_ACCESS_KEY: circle_test
           BUCKETEER_BUCKET_NAME: circle_test
           CONNECTED_APP_CALLBACK_URL: circle_test
           CONNECTED_APP_CLIENT_ID: circle_test
           CONNECTED_APP_CLIENT_SECRET: circle_test
-          DATABASE_URL: postgres://ubuntu@localhost:5432/circle_test
+          DATABASE_URL: postgres://ubuntu:circle_test@localhost:5432/circle_test
           DB_ENCRYPTION_KEY: s38Tpa-rzNJsDwmn_bNuUK4T1zaNG9j1J9u1Yrk9P9c=
           DJANGO_DEBUG: False
           DJANGO_HASHID_SALT: test_hash_id
           DJANGO_SECRET_KEY: test_secret_key
           DJANGO_SETTINGS_MODULE: config.settings.test
           GITHUB_TOKEN: circle_test
           SECURE_SSL_REDIRECT: False
-      - image: circleci/redis:latest
-      - image: circleci/postgres:latest
+      - image: circleci/redis:5.0
+      - image: circleci/postgres:10.6
```

Comment:
I note that locally we're defining postgres 10.6, but staging is running v11. @davisagli Is prod running v12?

Reply:
@jgerigmeyer Production has postgres 10.13 and redis 4.0.14. Staging has postgres 10.12 and redis 4.0.14

Reply:
Ah, okay. In that case I can revert the changes in https://github.com/oddbird/MetaDeploy/pull/24

Reply:
...unless you think we should just migrate the prod/staging envs?

Reply:
We should, but I'll need to plan that for off-hours, and I'm not aware of a particular reason to prioritize it.